rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_20.json

```diff
@@ -0,0 +1,572 @@
+                                                    "cooling_system_type": "FLUID_LOOP",
+                                                    "chilled_water_loop": "Secondary CHW Loop 1"
+                                                },
+                                                "preheat_system": {
+                                                    "id": "Preheat Coil 1",
+                                                    "heating_system_type": "FLUID_LOOP",
+                                                    "hot_water_loop": "Boiler Loop 1"
+                                                },
+                                                "fan_system": {
+                                                    "id": "VAV Fan System 1",
+                                                    "fan_control": "VARIABLE_SPEED_DRIVE",
+                                                    "supply_fans": [
+                                                        {
+                                                            "id": "Supply fan 1",
+                                                            "specification_method": "SIMPLE",
+                                                            "design_electric_power": 100,
+                                                            "design_airflow": 1000
+                                                        }
+                                                    ],
+                                                    "return_fans": [
+                                                        {
+                                                            "id": "Return fan 1",
+                                                            "specification_method": "SIMPLE",
+                                                            "design_electric_power": 50,
+                                                            "design_airflow": 800
+                                                        }
+                                                    ],
+                                                    "relief_fans": [
+                                                        {
+                                                            "id": "Relief fan 1",
+                                                            "specification_method": "SIMPLE",
+                                                            "design_electric_power": 50
+                                                        }
+                                                    ],
+                                                    "exhaust_fans": [
+                                                        {
+                                                            "id": "Exhaust fan 1",
+                                                            "specification_method": "SIMPLE",
+                                                            "design_electric_power": 0
```

Comment:
```percent_of_total_exhaust_p``` value  becomes 0 because ```design_electric_power``` value is 0. In this case,  total exhaust fan power should also be 0 to make this case pass. Could you address this?

Reply:
Corrected.